packages/stat-logger/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@agoric/stat-logger",
+  "version": "0.1.0",
```

Comment:
Can we make this `private: true`? Or does it need to be published?

Reply:
It only needs to be public if `swingset-runner` is too.  But if we want to publish _that_, we'll need to either need to have `stat-logger` be public or come up with some way to have a package optionally depend on another (which I presume can be done, but I don't know the recipe).

Reply:
Added as issue #678